elliottlib/cli/find_bugs_cli.py

```diff
@@ -123,49 +137,74 @@ def find_bugs_cli(runtime, advisory, default_advisory_type, mode, check_builds,
 
 \b
     $ elliott --group=openshift-4.6 --mode qe
+
+\b
+    $ elliott --group=openshift-4.6 --mode blocker --report
 """
+    advisory_attach_flags = sum(map(bool, [advisory, default_advisory_type, into_default_advisories]))
+
     if mode != 'list' and len(id) > 0:
         raise click.BadParameter("Combining the automatic and manual bug attachment options is not supported")
 
     if mode == 'list' and len(id) == 0:
         raise click.BadParameter("When using mode=list, you must provide a list of bug IDs")
 
-    if mode == 'payload' and not len(from_diff) == 2:
-        raise click.BadParameter("If using mode=payload, you must provide two payloads to compare")
+    if mode == 'diff' and not len(from_diff) == 2:
+        raise click.BadParameter("If using mode=diff, you must provide two payloads to compare")
 
-    if sum(map(bool, [advisory, default_advisory_type, into_default_advisories])) > 1:
+    if advisory_attach_flags > 1:
         raise click.BadParameter("Use only one of --use-default-advisory, --add, or --into-default-advisories")
 
-    if mode == 'qe' and sum(map(bool, [advisory, default_advisory_type, into_default_advisories])) > 0:
-        raise click.BadParameter("--mode=qe does not operate on an advisory. Do not specify any of `--use-default-advisory`, `--add`, or `--into-default-advisories`")
+    if mode in ['qe', 'blocker'] and advisory_attach_flags > 0:
+        raise click.BadParameter("Mode does not operate on an advisory. Do not specify any of "
+                                 "`--use-default-advisory`, `--add`, or `--into-default-advisories`")
 
     runtime.initialize()
     bz_data = runtime.gitdata.load_data(key='bugzilla').data
     bzapi = bzutil.get_bzapi(bz_data)
 
+    # Some bugs should goes to CPaaS so we should ignore them
+    m = re.match(r"rhaos-(\d+).(\d+)",
+                 runtime.branch)  # extract OpenShift version from the branch name. there should be a better way...
+    if not m:
+        raise ElliottFatalError(f"Unable to determine OpenShift version from branch name {runtime.branch}.")
+    major_version = int(m[1])
+    minor_version = int(m[2])
+
     if default_advisory_type is not None:
         advisory = find_default_advisory(runtime, default_advisory_type)
 
-    if mode == 'sweep' or mode == 'qe':
-        if mode == 'qe':
-            status = ['MODIFIED']
+    if mode in ['sweep', 'qe', 'blocker']:
+        if not cve_trackers:
+            if mode == 'blocker':
+                cve_trackers = True
+            else:
+                cve_trackers = False
```

Comment:
Seems a bit verbose, how about:
```suggestion
            cve_trackers = mode == 'blocker'
```

Reply:
Not sure about that, I prefer the first one since the branching makes it very clear that cve_trackers value varies by mode. The value could be bool, or string doesn't matter, the structure would be: `if not param: if mode == 'X': param = Y`

Reply:
Also this is soon to be refactored into individual function for each mode, so we would get rid of this anyway :)